packages/dds/sequence/api-report/sequence.legacy.public.api.md

```diff
@@ -4,34 +4,4 @@
 
 ```ts
 
-export { BaseSegment }
-
-export { InteriorSequencePlace }
-
-export { ISegment }
-
-export { LocalReferencePosition }
-
-export { MapLike }
-
-export { Marker }
-
-export { MergeTreeDeltaType }
-
-export { PropertySet }
-
-export { ReferencePosition }
-
-export { ReferenceType }
-
-export { reservedMarkerIdKey }
-
-export { SequencePlace }
-
-export { Side }
-
-export { TextSegment }
-
-export { TrackingGroup }
```

Comment:
These seem to be `@legacy + @alpha` which has its own api report, so correct they disappear here?